src/status_im/contexts/wallet/home/view.cljs

```diff
@@ -105,6 +105,7 @@
                                [multiple-accounts? first-account-address])
         on-swap-press         (rn/use-callback
                                (fn []
+                                 (rf/dispatch [:wallet/clean-send-data])
```

Comment:
This is okay decision but it would be ideal to find the place where state is not cleaned after exiting bridge state and call the cleanup there. Because cleanup of bridge shouldn't be a business for swap like it is in this fix.

Reply:
We are calling this event also for send action and also swap in account view. the scope of this issue I believe this changes would be enough @vkjr.